web3.md

```diff
@@ -630,13 +630,15 @@ eth_sendTransaction
            ...
          </account>
 
-    syntax JSON ::= #generateException( String, Int, Bytes ) [function]
- // -------------------------------------------------------------------
-    rule #generateException(TXHASH, PCOUNT, RD) => { "message": "VM Exception while processing transaction: revert",
+    rule <k> #loadNonce _ _ => 0 ... </k> [owise]
```

Comment:
Why is this here? Why does it put an `Int` on the `<k>` cell when the normal rule for `#loadNonce` doesn't put anything on the cell?

Reply:
This was a typo.